mne/viz/raw.py

```diff
@@ -786,9 +786,8 @@ def _prepare_mne_browse_raw(params, title, bgcolor, color, bad_color, inds,
         size = tuple([float(s) for s in size])
 
     fig = figure_nobar(facecolor=bgcolor, figsize=size)
-    fig.canvas.set_window_title('mne_browse_raw')
+    fig.canvas.set_window_title(title)
     ax = plt.subplot2grid((10, 10), (0, 1), colspan=8, rowspan=9)
-    ax.set_title(title, fontsize=12)
```

Comment:
I see you set `title` above but not `fontsize` anywhere, does it still look okay?

Reply:
I moved the title from the canvas to the window title bar, not sure what you mean.

Reply:
Oh I didn't notice that difference. I think that's a nice change